package.json

```diff
@@ -31,5 +31,10 @@
     "babel-cli": "~6.14.0",
     "babel-plugin-add-module-exports": "~0.2.1",
     "babel-preset-es2015": "~6.14.0"
+  },
+  "jspm": {
+    "main": "src/index.js",
```

Comment:
There is a reason only the transpiled version is used.
Please follow the guidelines by only using ’lib/index.js’
There is no reason it wouldn't work with JSPM as this is not the only library use this system.
Development shouldn't be affected, I'm pretty sure you'll find a solution, as I do using npm.